tools/deploy_prow.sh

```diff
@@ -71,6 +73,12 @@ function launchMonitoring(){
   make -C config/prow/monitoring
 }
 
+function launchARC(){
+  kubectl apply -f https://github.com/actions/actions-runner-controller releases/download/v0.22.0/actions-runner-controller.yaml
+  kubectl create secret generic controller-manager -n actions-runner-system --from-literal=github_token=${PROW_OAUTH_TOKEN}
```

Comment:
note to self: double check that the `PROW_OAUTH_TOKEN` is the right variable to use

Reply:
I can confirm it should be the right one